File: lecture6/pset/sentimental-readability/readability.py
# TODO
from cs50 import get_string()

def main():
    # get the input from user
    text = get_string("Enter Text Here: ")
    # declaring variables
    letters = 0
    words = 0
    sentence = 0
    # loop to go through text
    for i in range(len(text)):
        if str.isalpha(text):
            letters += 1
        elif str.isspace(text):
            words += 1
        elif text[i] == '.' or text[i] == '?' or text[i] == '!':
            sentence += 1
    # 




if __name__ == "__main__":
    main()

File: lecture2/psets/readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main(argc, argv[])
{
    if (argc != 2)
    {
        printf("Usage: ./readability key")
        return
    }
    string Text = get_string("Text: ");
    int words = 0;
    int letters = 0;
    int sentence = 0;

    for (int i = 0; i < Text; i++)
    {
        if (isalpha(argv))
    }
}